### DISCUSSION TITLE
ManimML：製作Machine Learning的運動動畫

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#1035] [優秀Python套件]
專案網址：https://github.com/helblazer811/ManimML

ManimML提供常見機器學習概念的動畫和可視化(Visualization)呈現。

Visualization台灣叫「視覺化」，對岸叫「可視化」，我底叫喜歡可視化的叫法

使用這個專案，就可以把程式碼用動畫展現，像下面的例子

## 神經網絡
![](https://github.com/helblazer811/ManimML/raw/main/examples/media/VAEScene.gif)

## 卷積神經網絡
![](https://github.com/helblazer811/ManimML/raw/main/examples/media/CNNScene.gif)

## 生成對抗網絡
![](https://github.com/helblazer811/ManimML/raw/main/examples/media/GANScene.gif)

## VAE
![](https://github.com/helblazer811/ManimML/raw/main/examples/media/DisentanglementScene.gif)

**Comment by quan0715** [comment]
謝謝教授分享，終於知道那些YT上酷酷的機器學習動畫是怎麼搞出來的，又多了一個 ML 報告呈現利器，視覺化絕對是讓人們最直觀接受一些想法的大哉問，好的視覺化可以有效的把繁瑣的理論、公式、數據 用生動有趣的方式呈現出來。
 by the way 第三行 我底叫喜歡可視化的叫法 -> 我比叫喜歡可視化的叫法
